### PR TITLE
DOC-613: Anchor known issue in 5.4

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-613

Description of Changes:
* Document 5.4 known issue regarding anchor_top and anchor_bottom options.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: release-notes/release-notes54.md [hunk 1]
```diff
@@ -187,6 +188,34 @@ To develop and maintain a new spellchecking plugin based on the free {{site.prod
...
+### Top and bottom editor anchors appear in links autocomplete list when disabled
+
+#### Issue
+
+This issue affects the following options:
+
+- `anchor_top`
+- `anchor_bottom`
+
+When this options are set to `false`, the relevant anchors should not appear as autocomplete options in the link dialog. These anchors will appear in the autocomplete list when the relevant options are set to `false` in {{site.productname}} 5.4.0 and {{site.productname}} 5.4.1.
```
Comment (lnewson) [inline release-notes/release-notes54.md:208]: When "these"?

Comment (tylerkelly13) [reply release-notes/release-notes54.md]: fixed in: https://github.com/tinymce/tinymce-docs/pull/1613/commits/1e6c1d0833ea57c8f926e5903187f3b26e731827

Comment (lnewson) [reply release-notes/release-notes54.md]: Not fixed

File: release-notes/release-notes54.md [hunk 2]
```diff
@@ -187,6 +188,45 @@ To develop and maintain a new spellchecking plugin based on the free {{site.prod
...
+For example:
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  typeahead_urls: false
+});
+```
+
+For information on the `typeahead_urls` option, see:  [Advanced editing behaviors - `typeahead_urls`]({{site.baseurl}}/configure/advanced-editing-behavior/#typeahead_urls).
```
Comment (lnewson) [inline release-notes/release-notes54.md:228]: Double space after `see:` here

Comment (tylerkelly13) [reply release-notes/release-notes54.md]: fixed here: https://github.com/tinymce/tinymce-docs/pull/1613/commits/2049ecce753cb81b16d9cbad95d85de3dbb58639

**Pull request merged**